Rakefile

```diff
@@ -1,6 +1,89 @@
-require "bundler/gem_tasks"
-require "rspec/core/rake_task"
+require "fileutils"
+
+task :run_rspec do
+  sh %{ rspec --exclude-pattern "spec/dummy/**/*_spec.rb" spec }
+  sh %{ cd spec/dummy && rspec }
+end
+
+task :default => :run_rspec
+
+namespace :lint do
+
+  desc "Run Rubocop as shell"
+  task :rubocop do
+    cmd = "rubocop ."
+    sh cmd
```

Comment:
@dylangrafmyre, @samnang is right on this one. No reason to have separate lines.